src/Analyzers/Readme.md

```diff
@@ -0,0 +1,22 @@
+# Object Pool
+
+This directory contains sources for analyers used internally by ASP.NET Core.
+
+## Description
+
+- `Analyzers`: Contains analyzers for ASP.NET Core shipped as part of the Microsoft.NET.Sdk.Web.
+- `Microsoft.AspNetCore.Analyzer.Testing`: Contains types for writing analyzer tests.
+
+## Development Setup
+
+### Build
+
+To build this specific project from source, follow the instructions [on building the project](../../docs/BuildFromSource.md#step-3:-build-the-repo).
```

Comment:
@captainsafia I noticed the template README.md is now out of date after the change https://github.com/dotnet/aspnetcore/commit/70e7c40a5091e0a01c80aafe416f8f49b62129ed#diff-57a537bf86f9dd51ea61e7b1bacd3adcfe77e7bb051c2245257e9a2fbac06ed8.

Reply:
```suggestion
To build this specific project from source, follow the instructions [on building the project](../../docs/BuildFromSource.md#step-3-build-the-repo).
```

Reply:
Search and replace for this sounds good!